src/fsharp/FSharp.Core/array.fs

```diff
@@ -1270,6 +1270,96 @@ namespace Microsoft.FSharp.Collections
                 let count' = Operators.min count len
                 Microsoft.FSharp.Primitives.Basics.Array.subUnchecked 0 count' array
 
+        [<CompiledName("RemoveAt")>]
+        let removeAt (index: int) (source: 'T[]) : 'T[] =
+            checkNonNull "source" source
+            if index < 0 || index >= source.Length then invalidArg "index" "index must be within bounds of the array"
+            
+            let length = source.Length - 1
+            let result = Microsoft.FSharp.Primitives.Basics.Array.zeroCreateUnchecked length
+
+            for i in [0..length] do
```

Comment:
Remoce the braket no need to create lisr same for all methods

Reply:
Yes will be much faster without

Reply:
This one should be fixed for all the functions now